Add show subtask

theredcat opened the pull request:
Follow Up for https://github.com/AlJohri/asana-cli/pull/8 to add subtask print as well.

I've splitted thoses PR if you want to keep the simple show command without the subtask option.

theredcat committed:
Add show tasks command

theredcat committed:
add --subtasks to show tasks command

theredcat commented:
@AlJohri This one is the more complete PR and is ready for review. I'm using it to dump tasks from asana and put them is an Elasticsearch.

My process look like this :
```
asana list projects --workspace=xxx | 
    jq .name| 
    xargs -I% bash -c '
        asana show tasks --project="%" --section="Dev in progress" --subtasks | 
        jq -cs "<a very long jq script that create the ES indexing bulk>"
    ' | 
    curl -X POST -H "Content-Type: application/x-ndjson" --data-binary @- "http://elastic_url:9200/index_name/_doc/_bulk"
```

AlJohri commented:
@theredcat nice stuff! unfortunately this project doesn't have any tests and I don't have an asana account so I can't test this, but the pull request looks small and reviewing the code it looks correct to me

I'm happy to opt for the show tasks with subtasks option-- let's keep #9 and close #8?

Also can you do a minor [version](https://github.com/AlJohri/asana-cli/blob/master/asana_cli/version.py) bump to 0.0.4?

after that, LGTM!

AlJohri commented:
also consider updating the help text in the README's [CLI section](https://github.com/AlJohri/asana-cli#cli) to reflect the new command

theredcat committed:
Add documentation

theredcat committed:
Bump version

theredcat commented:
Yup this PR make #8 Obselete. I've orignally separated them to give you more options.

I've updated the doc, bumped the version and rebased my commits to keep them self-sufficients

theredcat commented:
Hi @AlJohri Could you take a look ?
